.github/dependabot.yml

```diff
@@ -4,6 +4,13 @@ updates:
   - package-ecosystem: "gradle"
     directory: "/"
     open-pull-requests-limit: 10
```

Comment:
Double checking -- with this grouping enabled, do you still want 10 as the limit?

Reply:
I probably will go back and increase the WSM limit for consistency (although currently those PRs require some hand-holding, so we may want a lower limit in the short-term).

Reply:
Sounds good.  I think I mentioned on Slack but we ran into trouble with auto-rebase on our TDR UI PRs with a higher limit (competition for a small number of integration namespaces), and I really wanted to get auto-rebase back :)